e2e/keywords/node.resource

```diff
@@ -0,0 +1,121 @@
+*** Settings ***
+Documentation     
+...               Keywords defined here can be used where this Keywords.resource in loaded.
+Library    ../libs/steps.py
+Resource    ../keywords/utility.resource
+
+*** Variables ***
+${POD_EVICTION_TIME_OUT}=    300
+
+*** Keywords ***
+The ${size} GB ${volume_type} volume with ${request_replica_count} replicas
+    ${volume_name} =    create_volume   ${size}   ${request_replica_count}   ${volume_type}
+    Set Test Variable    ${volume_name}
+    Set Test Variable    ${request_replica_count}
+    Get node index with replica
+    Get node index without replica
+
+Attach volume to the node ${node_number} with replica node
+    ${target_node_number} =    Evaluate    ${node_number}-1
+    ${volunme_attached_node} =    attach_volume   ${volume_name}   ${node_number_with_replica}[${target_node_number}]
+    ${non_volunme_attached_node} =    get_non_volunme_attached_node   ${volunme_attached_node}
+    Set Test Variable    ${volunme_attached_node}
+    Set Test Variable    ${non_volunme_attached_node}
+
+Attach volume to the node ${node_number} without replica node
+    ${target_node_number} =    Evaluate    ${node_number}-1
+    ${volunme_attached_node} =    attach_volume   ${volume_name}   ${node_number_without_replica}[${target_node_number}]
+    ${non_volunme_attached_node} =    get_non_volunme_attached_node   ${volunme_attached_node}
+    Set Test Variable    ${volunme_attached_node}
+    Set Test Variable    ${non_volunme_attached_node}
+
+Attach volume to the no replica node
+    ${volunme_attached_node} =    attach_volume_without_replica   ${volume_name}   ${request_replica_count}
```

Comment:
```suggestion
    ${volume_attached_node} =    attach_volume_without_replica   ${volume_name}   ${request_replica_count}
```

Reply:
Have fixed the typo: `volunme` -> `volume`. Thank you.